scripts/make_attribution.sh

```diff
@@ -44,12 +44,13 @@ source "$REPO_ROOT/scripts/attribution_helpers.sh"
 function build::attribution::generate(){
     cd $REPO_ROOT
     build::common::use_go_version "$GOLANG_VERSION"
-    go mod vendor
+    local -r GO=$(build::common::get_go_path "$GOLANG_VERSION")/go
```

Comment:
We may actually need to leave this one how it was and rely on the default go version installed.  The go-licenses call can get weird if the currently configured go is not the same version that go-licenses was installed with. On builder base we only install it with the default version of go, not all the other versions.

Reply:
Gotcha. Leaving this as is.